client/src/pages/Members/Members.tsx

```diff
@@ -9,40 +9,43 @@ import MemberDetails from '../../components/MemberDetails/MemberDetails';
 
 import { Button } from 'react-bootstrap';
 
-const Members = (props) => {
+//TODO: REMOVE ALL @ts-ignore comments
+
+const Members: React.FC<RouteComponentProps<{}>> = ({ history, location }) => {
     const { SHOWINGRESULTSFOR, SHOWALLMEMBERSBUTTON, SHOWINGMEMBERSTITLE } = useContext(UIcontext).dictionary.membersPage;
 
-    const [members, setMembers] = useState([]);
+    const [members, setMembers] = useState<RegisteredUserType[]>([]);
```

Comment:
Isn't it necessary to import type(s)? That's a convenient solution.

Reply:
Excellent observation!
NO type imports at all! ;o)
And the reason: files with '...name.**_d_**.ts' are imported automagically!